### ISSUE TITLE
Animation: Blur and shrink on delete/collapseContext

### ISSUE THREAD

**Issue by raineorshine** [cybersemics/em#2527]
- [x] Blocked by: #2526

## Steps to Reproduce

```
- a
- b
- c
```

Delete `b`.

## Current Behavior

`b` fades out according to the default unmount animation.

## Expected Behavior

The thought should fade out, blur, and shrink (scale3d) when the following commands are executed:

- Delete
- Collapse Context

**Comment by raineorshine** [comment]
@zukilover Could you limit the fade/shrink/blur animation to the `delete` and `collapseContext` commands as requested in the original issue? You'll have to open a new PR as I merged #2596 prematurely.
